docs/riskAuthentication.md

```diff
@@ -50,4 +51,30 @@ If the `Device Profiling Method` setting in your adapter configuration is set to
 var authnWidget = new PfAuthnWidget('https://localhost:9031', { divId: 'authnwidget', deviceProfileScript: './tmx_sdk_profiling.js' });
 authnWidget.init();
 ```
-If the `Device Profiling Method` setting in your adapter configuration is set to `Captured by this adapter` and `Device Profiling Script Source` is set to `ThreatMetrix Web`, no extra steps are needed.
\ No newline at end of file
+If the `Device Profiling Method` setting in your adapter configuration is set to `Captured by this adapter` and `Device Profiling Script Source` is set to `ThreatMetrix Web`, no extra steps are needed.
+
+## PingOne Risk Management Integration Kit
+
+### System requirements and dependencies
+
+* PingOne Risk Management Integration Kit 1.0 or later
+
+### Setup
+1. Copy the `fingerprint2-2.1.4.min.js` and `pingone_risk_management_adapter_device_profiling.js` files from the integration `.zip` file to a location that your application can access. 
+2. Add the following to your application's sign-on page. Adjust the path to the script files.
+```html
+<script type="text/javascript" src="fingerprint2-2.1.4.min.js"></script>
+<script type="text/javascript" src="pingone_risk_management_adapter_device_profiling.js"></script>
```

Comment:
This script should is not needed on the index.html page since it will be included as deviceProfileScript for `Captured by this adapter`. The script should just be added for `Captured by a previous adapter`.

Reply:
Thanks, updated. Also updated to match new file names.